### PR TITLE
feat: implement counter metrics

### PR DESCRIPTION
**Pull request by Abdulsametileri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,4 +98,19 @@ func consumeFn(message kafka.Message) error {
...
 | `sasl.username`                    | SCRAM OR PLAIN username                                                                                                               |                  |
 | `sasl.password`                    | SCRAM OR PLAIN password                                                                                                               |                  |
-| `logger`                           | If you want to custom logger                                                                                                          | info             |                          |
+| `logger`                           | If you want to custom logger                                                                                                          | info             |
+| `apiEnabled`                       | Enabled metrics                                                                                                                       | false            |
+| `apiConfiguration.port`            | Set API port                                                                                                                          | 8090             |
```
Comment (Abdulsametileri) [inline README.md:103]: 8080 is a docker-compose kowl port, so I needed to specify a different port, like 8090.

File: README.md [hunk 2]
```diff
@@ -98,4 +98,19 @@ func consumeFn(message kafka.Message) error {
...
+
+## Monitoring
+
+Kafka Konsumer offers an API that handles exposing several metrics.
+
+### Exposed Metrics
+
+| Metric Name                                    | Description                                   | Value Type |
+|------------------------------------------------|-----------------------------------------------|------------|
+| kafka_kronsumer_processed_messages_total       | Total number of processed messages.           | Counter    |
```
Comment (emreodabas) [inline README.md:115]: typo kronsumer :D

Comment (emreodabas) [reply README.md:115]: same as below

File: consumer.go [hunk 3]
```diff
@@ -22,6 +22,7 @@ type consumer struct {
 	once        sync.Once
 	messageCh   chan Message
 	quit        chan struct{}
+	apiShutdown chan struct{}
```
Comment (emreodabas) [inline consumer.go:25]: Do we really need another quit channel? Why don't we call the stop functions of all sub-started processes by listening only to the quit channel?

**Review by mhmtszr — APPROVED** [review]
LGTM.

**Pull request merged**